examples/bpk-component-slider/stories.js

```diff
@@ -40,11 +40,13 @@ export default {
           <Title />
           <ArgsTable of={PRIMARY_STORY} />
           <Markdown>
-            {
-              `[Please refer to react-slider's documentation for a full list of props](https://zillow.github.io/react-slider/).
-              **Note:** When you're representing non-integer values (eg time, dates) please ensure you use \`ariaLabel\` and \`ariaValuetext\` to ensure that assistive technologies will be able to understand the value better.
-              `
-            }
+            {`**Note**: The aria props are the values that will be passed to the thumb of the slider. If your slider is for times for instance you would likely pass something like the following to ensure the value of the thumb is read out in a formatted state rather than just the value of where the thumb is on the track. If no \`ariaValuetext\` is passed to the component the screen reader will read just the value of the thumb
+
+            ariaLabels={['From', 'To']}
+    ariaValuetext={[getSliderTime(finalSliderStart), getSliderTime(finalSliderEnd)]}
+
+          `}
+
```

Comment:
This looks nice when rendered
![Screenshot 2024-04-16 at 16 10 50](https://github.com/Skyscanner/backpack/assets/28352715/1a3f5a12-7e9a-4090-b221-b35e97c25ae8)